lib/tasks/jenkins.rake

```diff
@@ -13,7 +13,7 @@ begin
         ENV["CI_REPORTS"] = 'jenkins/reports/unit/'
         gem 'ci_reporter'
       end
-      minitest_plugins = [:pre_ci, 'webpack:try_compile', 'ci:setup:minitest']
+      minitest_plugins = [:pre_ci, 'ci:setup:minitest']
```

Comment:
should we also drop the `try_compile` task from webpack.rake? or is there still something that uses it?

Reply:
It's used here - looks like for the integration tests

https://github.com/theforeman/foreman/blob/8355fb06ececcf72a4b80c027cc8015c297d27b6/lib/tasks/test.rake#L21-L26

Reply:
I wonder if we actually run it again there but don't see it because of the `verbose(false)`.... might be interesting to try and remove that block as well and see if the tests still pass

Reply:
I like the removal and only included if needed. When it's needed, we can hard fail.

Reply:
Good call @tbrisker ! Looks like the task isn't needed at all as the integration tests do run webpack:compile on their own through the jenkins task